

Test 'TestInit.c'
-----------------
In file included from Source/src/DSP280x_PieCtrl.c:14:0,
                 from test/TestInit.c:3:
Source/src/DSP280x_Device.h:42:0: warning: "EINT" redefined
 #define  EINT   asm(" clrc INTM")
 
<command-line>:0:0: note: this is the location of the previous definition
In file included from Source/src/DSP280x_PieCtrl.c:14:0,
                 from test/TestInit.c:3:
Source/src/DSP280x_Device.h:43:0: warning: "DINT" redefined
 #define  DINT   asm(" setc INTM")
 
<command-line>:0:0: note: this is the location of the previous definition
In file included from Source/src/DSP280x_PieCtrl.c:14:0,
                 from test/TestInit.c:3:
Source/src/DSP280x_Device.h:44:0: warning: "ERTM" redefined
 #define  ERTM   asm(" clrc DBGM")
 
<command-line>:0:0: note: this is the location of the previous definition
In file included from Source/src/DSP280x_PieCtrl.c:14:0,
                 from test/TestInit.c:3:
Source/src/DSP280x_Device.h:46:0: warning: "EALLOW" redefined
 #define  EALLOW asm(" EALLOW")
 
<command-line>:0:0: note: this is the location of the previous definition
In file included from Source/src/DSP280x_PieCtrl.c:14:0,
                 from test/TestInit.c:3:
Source/src/DSP280x_Device.h:47:0: warning: "EDIS" redefined
 #define  EDIS   asm(" EDIS")
 
<command-line>:0:0: note: this is the location of the previous definition
Compiling TestInit_runner.c...
build/test/runners/TestInit_runner.c:1:0: error: cannot open build/gcov/out/TestInit_runner.gcno
 /* AUTOGENERATED FILE. DO NOT EDIT. */
 
ERROR: Shell command failed.
> Shell executed command:
'gcc.exe -g -fprofile-arcs -ftest-coverage -I"C:/Ruby30-x64/lib/ruby/gems/3.0.0/gems/ceedling-0.31.1/vendor/unity/src" -I"C:/Ruby30-x64/lib/ruby/gems/3.0.0/gems/ceedling-0.31.1/vendor/cmock/src" -I"build/test/mocks" -I"test" -I"test/support" -I"Source/src" -I"Source/src/Cmd" -I"Source/src/Libs" -DTEST -DCEEDLING -DUNITY_SUPPORT_64 -D__TMS320C2000__=1 -D__TMS320C28X__=1 -D__TMS320C28XX__=1 -D__PTRDIFF_T_TYPE__=long -D__SIZE_T_TYPE__="unsigned long" -D__WCHAR_T_TYPE__="unsigned int" -D__uint8_t=short -D__int8_t=short -D__asm(x) -D__cregister -DEALLOW -DEDIS -DEINT -DERTM -DDINT -D__interrupt -D__TI_EABI__ -Dasm(x) -Dbyte_peripheral -Dcregister= -Dfar -Dinterrupt= -DGCOV_COMPILER -DCODE_COVERAGE -c "build/test/runners/TestInit_runner.c" -o "build/gcov/out/TestInit_runner.o"'
> And exited with status: [1].

#<Thread:0x0000022959564b48 C:/Ruby30-x64/lib/ruby/gems/3.0.0/gems/ceedling-0.31.1/lib/ceedling/par_map.rb:7 run> terminated with exception (report_on_exception is true):
C:/Ruby30-x64/lib/ruby/gems/3.0.0/gems/ceedling-0.31.1/lib/ceedling/tool_executor.rb:88:in `exec': ShellExecutionException (ShellExecutionException)
	from C:/Ruby30-x64/lib/ruby/gems/3.0.0/gems/ceedling-0.31.1/lib/ceedling/generator.rb:99:in `generate_object_file'
	from C:/Ruby30-x64/lib/ruby/gems/3.0.0/gems/ceedling-0.31.1/plugins/gcov/gcov.rake:23:in `block in <top (required)>'
	from C:/Ruby30-x64/lib/ruby/gems/3.0.0/gems/rake-13.0.3/lib/rake/task.rb:281:in `block in execute'
	from C:/Ruby30-x64/lib/ruby/gems/3.0.0/gems/rake-13.0.3/lib/rake/task.rb:281:in `each'
	from C:/Ruby30-x64/lib/ruby/gems/3.0.0/gems/rake-13.0.3/lib/rake/task.rb:281:in `execute'
	from C:/Ruby30-x64/lib/ruby/gems/3.0.0/gems/rake-13.0.3/lib/rake/task.rb:219:in `block in invoke_with_call_chain'
	from C:/Ruby30-x64/lib/ruby/gems/3.0.0/gems/rake-13.0.3/lib/rake/task.rb:199:in `synchronize'
	from C:/Ruby30-x64/lib/ruby/gems/3.0.0/gems/rake-13.0.3/lib/rake/task.rb:199:in `invoke_with_call_chain'
	from C:/Ruby30-x64/lib/ruby/gems/3.0.0/gems/rake-13.0.3/lib/rake/task.rb:188:in `invoke'
	from C:/Ruby30-x64/lib/ruby/gems/3.0.0/gems/ceedling-0.31.1/lib/ceedling/task_invoker.rb:97:in `block in invoke_test_objects'
	from C:/Ruby30-x64/lib/ruby/gems/3.0.0/gems/ceedling-0.31.1/lib/ceedling/par_map.rb:10:in `block (2 levels) in par_map'
rake aborted!
ShellExecutionException: ShellExecutionException

Tasks: TOP => build/gcov/out/TestInit_runner.o
(See full trace by running task with --trace)
ERROR: Ceedling Failed
